component-saml/src/main/java/com/synopsys/integration/alert/component/saml/SAMLGlobalConfigModel.java

```diff
@@ -0,0 +1,69 @@
+package com.synopsys.integration.alert.component.saml;
+
+import com.synopsys.integration.alert.common.rest.model.ConfigWithMetadata;
+
+import java.util.Optional;
+
+public class SAMLGlobalConfigModel extends ConfigWithMetadata {
+    private Boolean enabled;
+    private Boolean forceAuth;
+    private String metadataUrl;
+    private String metadataFilePath;
+    private String entityId;
+    private String entityBaseUrl;
+    private Boolean requireAssertionsSigned;
+    private String roleAttributeMapping;
+
+    public SAMLGlobalConfigModel () {
+        // For serialization
+    }
+
+    // Required
+    public SAMLGlobalConfigModel (String id, String name) {
+        super(id, name);
+    }
+
+    public SAMLGlobalConfigModel (
+        String id,
+        String name,
+        String createdAt,
+        String lastUpdated,
+        Boolean enabled,
+        Boolean forceAuth,
+        String metadataUrl,
+        String metadataFilePath,
+        String entityId,
+        String entityBaseUrl,
+        Boolean requireAssertionsSigned,
+        String roleAttributeMapping
+    ) {
+        this(id, name);
+        this.enabled = enabled;
+        this.forceAuth = forceAuth;
+        this.metadataUrl = metadataUrl;
+        this.metadataFilePath = metadataFilePath;
+        this.entityId = entityId;
+        this.entityBaseUrl = entityBaseUrl;
+        this.requireAssertionsSigned = requireAssertionsSigned;
+        this.roleAttributeMapping = roleAttributeMapping;
+
+        setCreatedAt(createdAt);
+        setLastUpdated(lastUpdated);
+    }
+
+    public Optional<Boolean> getEnabled() { return Optional.ofNullable(enabled); }
```

Comment:
Minor thing, I believe our code style for the public accessors is to still put them across multiple lines even if theres only a single line return. Typically the IDE will collapse them into a single line that can be expanded.